lib/SILOptimizer/Mandatory/TFUtilities.h

```diff
@@ -239,6 +239,14 @@ struct GraphGlobalConfiguration {
   /// LLVM Builtin type like Builtin.f32) into the TensorFlow TF_DataType value.
   unsigned convertSwiftTypeToTF(Type ty);
 
+  /// `ty` must be a valid TensorFlow element type "T", like Builtin.Int32. Turn
+  /// it into a TensorHandle<T> type.
+  SILType convertToTensorValueType(Type ty, const ASTContext& ctx);
```

Comment:
I feel having the overloaded names here could cause potential confusion. How about rename to sth like: convertElementTypeToTensorValueType()?

(now that we are moving this method to a more public API, it'd be useful to scrutinize the naming a bit more)

Reply:
Good point. Changed to convertElementTypeToTensorValueType.